### PR TITLE
FIX: Typos and remove default URL

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/v6jctrxzb5kdm57t3ow4wz4gve?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10922)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/kqezhzyyr4pk2a7dukr5w3na5i?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/10923)

**Review by Overtorment — APPROVED** [review]

File: loc/da_dk.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "header": "indstillinger",
     "language": "Sprog",
     "lightning_settings": "Lightning settings",
-    "lightning_settings_explain": "To connect to your own LND node please install LndHub and put its URL here in settings. Leave blank to use default ndHub\n (lndhub.io)",
+    "lightning_settings_explain": "To connect to your own LND node please install LndHub and put its URL here in settings. Leave blank to use default ndHub\n ",
     "password": "Adgangskode",
```
[the remark this answers is not in the document]
Comment (Overtorment) [reply loc/da_dk.json:61]: `Adgangskode` might be someone's password lol. looks random enough

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**